backend/include/Helpers.hpp

```diff
@@ -66,8 +91,7 @@ class NapiHelpers {
 
         callbackRef->NonBlockingCall(
             [eventName, data, data2](Napi::Env env, Napi::Function jsCallback) {
-                LOG_TRACE_L1(quill::get_logger("trackaudio_logger"),
-                    "Event name: {}, data: {}, data2: {}", eventName, data, data2);
+                TRACK_LOG_TRACE_L1("Event name: {}, data: {}, data2: {}", eventName, data, data2);
```

Comment:
I added `TRACK_LOG_TRACE_L1` as a macro since I needed it and it was missing. Updated this one place where `LOG_TRACE_L1` was being used directly.